packages/salesforcedx-apex-debugger/src/constants.ts

```diff
@@ -9,6 +9,8 @@ export const DEFAULT_INITIALIZE_TIMEOUT_MS = 10000;
 export const DEFAULT_STREAMING_TIMEOUT_MS = 14400;
 export const DEFAULT_CONNECTION_TIMEOUT_MS = 20000;
 export const DEFAULT_LOCK_TIMEOUT_MS = 10000;
+export const DEFAULT_IDLE_TIMEOUT_MS = 1200000;
```

Comment:
20 minutes is the idle timeout.

Reply:
Ah, ok. I assumed it was an hour because that was what we used in Eclipse. You can change "for an hour" to "for 20 minutes" in my suggested message.